### ISSUE TITLE
Higan does not take controller input when in full screen mode

### ISSUE THREAD

**Issue by karai17** [higan-emu/higan#57]
I am using Higan v110 on Windows 10 x86_64. I created a new Famicom system, set up a controller, etc. When I load up my ROM, everything seems to be fine and I can play the game. When I hit F11 (my toggle fullscreen key), my controller no longer provides any input. When I hit F11 again to return to windowed mode, I am able to continue playing.

I also tried with the Super Famicom system as well, same issue.

**Comment by ageekhere** [comment]
I have the same issue

**Comment by leftereislazar** [comment]
Had the same issue.
Go to the input settings and change Focus Loss to Allow input

**Comment by karai17** [comment]
Yup, that fixed it! thanks

---
Landon Manning
lmanning17@gmail.com

**Comment by leftereislazar** [comment]
just remember that now you allow input in the background in all situations, if you change window for example (i think)

**Comment by karai17** [comment]
yup! this is a reasonable work around, but hopefully a proper fix can be implemented~
